### ISSUE TITLE
[BUG] Weather Widgets do not display icons

### ISSUE THREAD

**Issue by ghalpha** [Lissy93/dashy#1205]
### Environment

Self-Hosted (Docker)

### System

UNRAID 6.12.0 RC6

### Version

2.1.1

### Describe the problem

Weather widget does not display any icons. I have confirmed my API call does not show a 401 error, and all data does display but only the icons do not. I have tested this on multiple browsers as well with the same result. I'll admit my web troubleshooting skills might not be up to where others are, but I am pretty sure I have everything right. I did try to dig in with the inspect element and try to figure out what was going on but honestly couldn't find an issue other than I don't see a reference to an actual image anywhere like the OpenWeather API document explains. I'm unfortunately more comfortable with desktop development than web development so this is new and uncharted territory for me. 

The basis I started with was the dashboard example and worked from there. Example here (https://gist.githubusercontent.com/Lissy93/000f712a5ce98f212817d20bc16bab10/raw/b08f2473610970c96d9bc273af7272173aa93ab1/Example%252010%2520-%2520Start%2520Page%2520conf.yml)

What I am seeing,
![image](https://github.com/Lissy93/dashy/assets/37425390/370b602f-107f-4f56-a3a0-c8d89d4f72d3)

My conf.yml snip of the section with weather,

sections:
  - name: Today
    icon: far fa-smile-beam
    displayData:
      collapsed: false
      hideForGuests: true
    widgets:
    - type: clock
      options:
      timeZone: America/Chicago
      format: en-US
      hideDate: false
    - type: weather
      options:
        apiKey: omitted
        city: omitted
        units: imperial
        hideDetails: false
    - type: weather-forecast
      options:
        apiKey: omitted
        city: omitted
        numDays: 6
        units: imperial

From the browser inspect on the icon element, it looks like the icon would be the correct one, but nothing displays. 
![image](https://github.com/Lissy93/dashy/assets/37425390/f1df24c8-0dfd-4068-9d7f-d7ff0cd94cdd)

I am hoping that I have done enough investigation and provided the needed information. I have spent a few days experimenting/testing and I am ready to ask for some help. I couldn't find where this issue had been documented or resolved in the past anywhere. 

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy
- [X] You've checked that this [issue hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide ![important](https://img.shields.io/badge/Important!-F6094E)
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by Lissy93** [comment]
Can you show the content of the browser console? [see how](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#how-to-open-browser-console)

I've just tested with that config file, and seems to work okay. Weather icons are loaded as a font, so it's something up with how your browsers displaying them.

**Comment by ghalpha** [comment]
So that makes more sense, it looks like I am having a resource issue for some reason. 
![image](https://github.com/Lissy93/dashy/assets/37425390/29f5d069-4511-4caa-95e3-b590bd8bd96d)

When following the font link it is missing as well. I'm digging around the docker container and I don't see that path or file and I pulled the latest image again already. Trying to figure out why that is missing. 

After investigating the issue further I found that the way I had my path to map my pages between the host and the container was less than ideal. When I looked through the /public/ folder in comparison to the repository it was obvious the volume mapping was the issue. Once I remapped the volumes in a different way I managed to get it to load. 

For future reference based on the information you have shared, attempting to map /app/public to a host path was my error. So to manage pages externally outside of the conf.yml. I created a mapped volume in /app/public/pages and then put my additional pages there. This is so I can still use code server on the host but easily interact with the other parts of Dashy. 

Thanks for the information and the step I missed in my troubleshooting.
